src/app/filecoin-plus/allocators/utils/getAllocatorUrlList.ts

```diff
@@ -27,3 +22,13 @@ function isValidAllocatorFile(file: AllocatorFileMetaDataBase) {
   const allocatorFilePattern = /^\d{3,4}\.json$/
   return allocatorFilePattern.test(file.name)
 }
+
+async function getAllocatorList() {
```

Comment:
Move before `extractAllocatorUrls`?

Reply:
I also tried to use Octokit in the other fetch function but couldn't figure out a quick implementation. We might want to do it at some point though :)

Reply:
>I also tried to use Octokit in the other fetch function but couldn't figure out a quick implementation. We might want to do it at some point though :)

From what I understand, using Octokit for the second case (fetching file contents directly from a URL) isn’t a good use case. 

> Octokit is ideal for interacting with the GitHub API, such as fetching repository metadata or directory contents, but it doesn’t handle raw file downloads. Since the raw file URLs are direct HTTP requests, the fetch function is more appropriate in this scenario. Octokit simplifies API interactions, while fetch is best suited for retrieving raw file content. Therefore, keeping both methods makes sense for these distinct use cases.

Reply:
Good to know!